src/components/Message/MessageText.js

```diff
@@ -106,9 +116,14 @@ const MessageTextComponent = (props) => {
         )}
       </div>
       <MessageOptions
-        {...props}
-        {...customOptionProps}
         onReactionListClick={onReactionListClick}
+        initialMessage={props.initialMessage}
+        message={message}
+        messageWrapperRef={props.messageWrapperRef}
+        threadList={props.threadList}
+        getMessageActions={props.getMessageActions}
+        handleFlag={props.handleFlag}
```

Comment:
🥇

Reply:
hehe. It broke some types. We need some adjustments in other places so I'll tackle this on a different PR. I reverted here: https://github.com/GetStream/stream-chat-react/pull/452/commits/9aa529462d7187c695586248842d5811880441ca.